cypress/integration/upload-download.spec.ts

```diff
@@ -0,0 +1,37 @@
+import {UploadFilePage, DownloadFilePage} from "../page/index";
+const fileName = "terminal-screen.png";
+const textTest =
+  "Lorem Ipsum is simply dummy text of the printing and typesetting indust";
+const pdfTest = "the user navigates to the upload and download file page should";
+
+describe("the user navigates to the upload and download file page should", () => {
```

Comment:
at this time we are not navigating to both pages yet, that is performed in each `it`, the title of this describe should say something more generic, for example, "The user navigates" and then each `it` will describe if it is for the download or for the upload. Another approach can be create another describe for the download file test.